src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -354,9 +354,7 @@ Structured contexts are typically set automatically.
 In addition to the structured context that Sentry understands, you can send arbitrary key/value pairs of data which the Sentry SDK will store alongside the event. These are not indexed, and the Sentry SDK uses them to add additional information about what might be happening:
 
 ```javascript
-Sentry.configureScope((scope) => {
-  scope.setExtra("character_name", "Mighty Fighter");
-});
+Sentry.setExtra("character_name", "Mighty Fighter");
```

Comment:
So this is something new and we might want to upgrade other JS docs.
`setExtra`, `setTag` and `setUser` can now be used without `configureScope`.

Reply:
Made a note of this. :)